src/components/Modal/VotingPowerDelegationDetail/VotingPowerDelegationDetail.css

```diff
@@ -27,12 +111,44 @@
 }
 
 .VotingPowerDelegationDetail__Content .Info--expanded {
-  max-height: 2000px;
+  max-height: 3000px !important;
```

Comment:
This needs a rework

Reply:
It's getting taller every day, amazing how fast classes grow 🥲